README.md

```diff
@@ -245,9 +245,9 @@ The parameters needed for each command are appended to the base URL, separated w
 
 | URL                                       | Response                                                                                                                | Source                                                                                           |
 | ----------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
-| `http://localhost:8000`                   | web page located at `public/index.html`                                                                                 | [:octocat:](https://github.com/Satyam/firmata-http-bridge/blob/main/src/server.ts#L53-L58)       |
-| `http://localhost:8000/something.txt`     | text file located at `public/something.txt`                                                                             | [:octocat:](https://github.com/Satyam/firmata-http-bridge/blob/main/src/server.ts#L60-L65)       |
-| `http://localhost:8000/dist/index.js`     | Javascript file located at `dist/index.js`                                                                              | [:octocat:](https://github.com/Satyam/firmata-http-bridge/blob/main/src/server.ts#L34-L39)       |
+| `http://localhost:8000`                   | web page located at `public/index.html`                                                                                 | [:octocat:](https://github.com/Satyam/firmata-http-bridge/blob/main/src/server.ts#L60-L65)       |
```

Comment:
Hello, just testing the comment feature :)
Andrea

Reply:
Good!  I wouldn't have known it was you.  That is the advantage of being an old-timer.  Nowadays, nobody can pick an email such as Daniel.Barreiro@gmail.com or a username like Satyam

Reply:
haha yep, it is hard nowadays to get easy emails and usernames like yours ;)